infra/aws/terraform/prow-build-cluster/resources/README.md

```diff
@@ -0,0 +1 @@
+[Installing AWS Node Termination Handler](https://github.com/aws/aws-node-termination-handler/tree/main/config/helm/aws-node-termination-handler#installing-the-chart)
```

Comment:
Would you mind extending this README with some more information about resources located in this directory?

Reply:
I added some context and moved it to a dedicated folder.